libkbfs/block_retrieval_queue.go

```diff
@@ -6,19 +6,19 @@ package libkbfs
 
 import (
 	"container/heap"
+	"io"
 	"sync"
 
 	"golang.org/x/net/context"
 )
 
 const (
-	defaultBlockRetrievalWorkerQueueSize int = 100
+	defaultBlockRetrievalWorkerQueueSize int = 20
```

Comment:
I had it reduced because it was annoying to scroll through 100 goroutines in tests, but I guess I could put it back.